File: TotalWavinessofNumbersInRange_I.js
var totalWaviness = function(num1, num2) {
    let total = 0;
    for (let n = num1; n <= num2; n++) {
        const s = String(n);
        for (let i = 1; i < s.length - 1; i++) {
            if ((s[i] > s[i-1] && s[i] > s[i+1]) ||
                (s[i] < s[i-1] && s[i] < s[i+1]))
                total++;
        }
    }
    return total;
};

const num1 = 120
const num2 = 130
const result = totalWaviness(num1, num2);
console.log("Total Waviness:", result);

File: TotalWavinessofNumbersInRange_I.cpp
#include <iostream>
#include <string>
using namespace std;

class TotalWavinessofNumbersInRange_I {
public:
    int totalWaviness(int num1, int num2) {
        int total = 0;
        for (int n = num1; n <= num2; n++) {
            string s = to_string(n);
            int len = s.size();
            for (int i = 1; i < len - 1; i++) {
                if ((s[i] > s[i-1] && s[i] > s[i+1]) ||
                    (s[i] < s[i-1] && s[i] < s[i+1]))
                    total++;
            }
        }
        return total;
    }
};

int main() {
    TotalWavinessofNumbersInRange_I solution;
    int num1 = 120, num2 = 130;
    int result = solution.totalWaviness(num1, num2);
    cout << "Total Waviness: " << result << endl;
    return 0;
}